core/capabilities/ccip/configs/solana/chain_writer.go

```diff
@@ -55,46 +55,51 @@ func getCommitMethodConfig(fromAddress string, routerProgramAddress string, comm
 				IsWritable: true,
 			},
 			chainwriter.PDALookups{
-				Name: "RouterReportAccount",
-				PublicKey: chainwriter.AccountConstant{
-					Address:    routerProgramAddress,
-					IsSigner:   false,
-					IsWritable: false,
-				},
+				Name:      "CommitReport",
+				PublicKey: offrampAddressConstant(offrampProgramAddress),
 				Seeds: []chainwriter.Seed{
 					{Static: []byte("commit_report")},
 					{Dynamic: chainwriter.AccountLookup{Location: merkleRootChainSelector}},
-					{Dynamic: chainwriter.AccountLookup{
-						Location: "Info.MerkleRoots.MerkleRoot",
-					}},
+					{Dynamic: chainwriter.AccountLookup{Location: merkleRoot}},
 				},
 				IsSigner:   false,
-				IsWritable: false,
+				IsWritable: true,
```

Comment:
How do we figure out for each lookup, this field is true or false?

Reply:
I checked the [IDL](https://github.com/smartcontractkit/chainlink-ccip/blob/main/chains/solana/contracts/target/idl/ccip_offramp.json) for these fields for each account. Sorry realized I didn't share that link with you offline.